db/schema.rb

```diff
@@ -0,0 +1,141 @@
+    t.index ["course_id"], name: "index_course_users_on_course_id"
+    t.index ["user_id"], name: "index_course_users_on_user_id"
+  end
+
+  create_table "courses", options: "ENGINE=InnoDB DEFAULT CHARSET=utf8", force: :cascade do |t|
+    t.string "name"
+    t.text "content"
+    t.date "start_date"
+    t.boolean "status", default: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
+  create_table "histories", options: "ENGINE=InnoDB DEFAULT CHARSET=utf8", force: :cascade do |t|
+    t.bigint "user_id"
+    t.bigint "course_subject_id"
+    t.string "title"
+    t.string "content"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["course_subject_id"], name: "index_histories_on_course_subject_id"
+    t.index ["user_id"], name: "index_histories_on_user_id"
+  end
+
+  create_table "subjects", options: "ENGINE=InnoDB DEFAULT CHARSET=utf8", force: :cascade do |t|
+    t.string "name"
+    t.text "details"
+    t.integer "duration_default"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
+  create_table "tasks", options: "ENGINE=InnoDB DEFAULT CHARSET=utf8", force: :cascade do |t|
+    t.string "title"
+    t.text "content"
+    t.bigint "subject_id"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["subject_id"], name: "index_tasks_on_subject_id"
+  end
+
+  create_table "user_subjects", options: "ENGINE=InnoDB DEFAULT CHARSET=utf8", force: :cascade do |t|
+    t.bigint "user_id"
+    t.bigint "subject_id"
+    t.date "finish_date"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["subject_id"], name: "index_user_subjects_on_subject_id"
+    t.index ["user_id", "subject_id"], name: "index_user_subjects_on_user_id_and_subject_id", unique: true
+    t.index ["user_id"], name: "index_user_subjects_on_user_id"
+  end
+
+  create_table "user_tasks", options: "ENGINE=InnoDB DEFAULT CHARSET=utf8", force: :cascade do |t|
+    t.bigint "user_id"
+    t.bigint "course_subject_task_id"
+    t.string "url_excel_file"
+    t.boolean "approved"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["course_subject_task_id"], name: "index_user_tasks_on_course_subject_task_id"
+    t.index ["user_id", "course_subject_task_id"], name: "index_user_tasks_on_user_id_and_course_subject_task_id", unique: true
+    t.index ["user_id"], name: "index_user_tasks_on_user_id"
+  end
+
+  create_table "users", options: "ENGINE=InnoDB DEFAULT CHARSET=utf8", force: :cascade do |t|
+    t.string "name"
+    t.string "email"
+    t.string "password"
+    t.string "password_digest"
+    t.string "gradution"
+    t.string "university"
+    t.string "address"
+    t.integer "sex", default: 1
+    t.string "role"
+    t.boolean "admin"
```

Comment:
line này có cần ko

role thì là kiểu int nhé và có giá trị default luôn -> chú tìm hiểu `enum in rails `giúp chị